src/base.ts

```diff
@@ -29,6 +29,10 @@ export abstract class AutocompleteBase extends Command {
     }
   }
 
+  public getSetupEnvVar(shell: string): string {
+    return `${this.cliBinEnvVar}_AC_${shell.toUpperCase()}_SETUP_PATH`
+  }
+
```

Comment:
I was building this string multiple times (and is also used in the output script) so made it a method in the base class.